Services/DirectoryService.cs

```diff
@@ -627,6 +631,72 @@ public Document Save(string name, string? sourceSubDirectory, string? targetSubD
             return GetDocument(TargetDirectory.Outbox, targetSubDirectory, newName ?? name);
         }
 
+        /// <summary>
+        /// Creates document in the inbox directory.
+        /// </summary>
+        /// <param name="name">The name.</param>
+        /// <param name="file">The File.</param>
+        public void Create(string name, IFormFile file)
+        {
+            string inboxDirectory = GetTargetDirectoryPath(TargetDirectory.Inbox);
+            string uploadFilePath = Path.Combine(inboxDirectory, name);
+
+            // Convert jpg to PDF
+            if (file.ContentType == "image/jpeg")
+            {
+                using (var imageStream = File.Create(uploadFilePath))
+                {
+                    file.CopyTo(imageStream);
+                }
+
+                using var magickimage = new MagickImage(uploadFilePath);
+                // ImageMagick docs say 40% should work for most images
+                magickimage.Deskew(new Percentage(40d));
+                magickimage.Enhance();
+                magickimage.Write(uploadFilePath);
+
+                ImageData imageData = ImageDataFactory.Create(uploadFilePath);
+                File.Delete(uploadFilePath);
+
+                string pdfFileName = name.Split(".").First() + ".pdf";
```

Comment:
I think this needs to handle filenames with multiple periods. Something like
`name.Substring(0, name.LastIndexOf('.'))`